ces_admin/views.py

```diff
@@ -22,6 +33,80 @@ def ces_logout(request):
     logout(request)
     return HttpResponseRedirect('/')
 
-@login_required(login_url='/ces-login/')
-def ces_admin(request):
-    return render(request, 'ces_admin/ces-admin.html')
\ No newline at end of file
+class DashboardView(TemplateView):
+    template_name = 'ces_admin/ces-dashboard.html'
+    
+    def get_context_data(self, **kwargs):
```

Comment:
There is a lot going on this method, which can be pulled into separate methods.

Reply:
I reorganized using a Mixin. What do you think?